src/modules/info-panel/info-panel.tag.html

```diff
@@ -1,27 +1,33 @@
-<info-panel class="viewer-info" style="height:60%;">
+<info-panel>
```

Comment:
also add aria-controls on the "info button" in the top toolbar and link it to this panel? Will also require `aria-expanded`.